### PR TITLE
Bug 1225333 - setAppearence when requesting unlock r=etienne

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3cdd662a58fc265466f56177bde128a836b3bc1e

File: apps/system/js/statusbar.js [hunk 1]
```diff
@@ -87,6 +87,8 @@
       window.addEventListener('iconshown', this);
       window.addEventListener('iconhidden', this);
       window.addEventListener('iconrendered', this);
+      window.addEventListener('lockscreen-request-unlock', this);
```
Comment (etiennesegonzac) [inline apps/system/js/statusbar.js]: nit: we should add the listener in `finishInit` with the related events

File: apps/system/test/unit/statusbar_test.js [hunk 2]
```diff
@@ -684,6 +684,10 @@ suite('system/Statusbar', function() {
       testEventThatShows.bind(this)('appchromeexpanded');
     });
 
+    test('appchromeexpanded', function() {
```
Comment (etiennesegonzac) [inline apps/system/test/unit/statusbar_test.js]: typo, we probably want to update the test name too :)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ee36ddc7858b79a52226ed58a51ada2a2a9ae9e4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1a9694273134870f15584df71cbe6a7d9af2a3d6

**Pull request merged**